client/components/App.jsx

```diff
@@ -112,16 +112,15 @@ class App extends Component {
       const { result: { items: calendars } } = await gapi.client.request({
         path: 'https://www.googleapis.com/calendar/v3/users/me/calendarList',
       });
-
-      const loggedInLC = this.state.loggedIn.toLowerCase();
-      let hirCalendarsSubscribed;
-      let hirCalendarLoggedIn;
+      const loggedInSplit = this.state.loggedIn.split(' ');
+      const loggedInFirstNameLC = loggedInSplit[0].toLowerCase();
+      const loggedInLastNameLC = loggedInSplit[loggedInSplit.length - 1].toLowerCase();
       let hirCalendarId;
       try {
-        hirCalendarsSubscribed = calendars.filter(
+        const hirCalendarsSubscribed = calendars.filter(
           ({ summary }) => summary.includes('HiR'));
-        hirCalendarLoggedIn = hirCalendarsSubscribed.find(
-          ({ summary }) => summary.includes(loggedInLC)) || hirCalendarsSubscribed[0];
+        const hirCalendarLoggedIn = hirCalendarsSubscribed.filter(
```

Comment:
Apparently .find is not supported in Internet Explorer, bummer.

Reply:
I hope we can trust Hack Reactor interviewers not to be using IE!